### ISSUE TITLE
User options view

### ISSUE THREAD

**Issue by devey** [rysk-finance/dynamic-hedging#350]
After buying or selling option user should be able to see details about his position.
This is mostly already implemented but just make sure any indicator or value user might need are there.

**Comment by devey** [comment]
Reassigned this to you @MeanBoyCousin, hope that's fine

**Comment by MeanBoyCousin** [comment]
Yeah, no worries. For the sake of narrowing the scope, I'm going to keep this relative to just the options chain. A new ticket for the dashboard can come after.

**Comment by devey** [comment]
@MeanBoyCousin I think it would be great if this first just worked on the dashboard (not sure what's the status of that). I think having it in the trading table is not a priority for internal testing

**Comment by MeanBoyCousin** [comment]
### The current status is as follows:

- The option chain currently displays only long positions. This can be updated once the graph work from @ugolino is finished regarding `options bought and sold`.
- The dashboard currently only shows long positions in one table. Again, once the `options bought and sold` work to the graph has been completed, we can migrate this table to show both long and short positions.
- It might be nice to separate the table in two (active & expired) or perhaps one table with filters. Possibly not required for the MVP though.

**Comment by devey** [comment]
Done in #448 (need to add some columns to provide more detailed overview of position like collateral type, vault health and P/L). Some of it I'll just do directly on Graph.
